### PR TITLE
retry a team load if we failed on proof error with forceRepoll=true

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

**Comment by mlsteele** [comment]
Is it your intent to must check merkle or to clear the whole cache?

**Comment by maxtaco** [comment]
here's another tyr

**Comment by mmaxim** [comment]
What is the status of this?

**Comment by maxtaco** [comment]
@mlsteele  you never gave me thumb?

**Comment by maxtaco** [comment]
why do that?

**Comment by mlsteele** [comment]
So that tests can catch the bugs that this works around.
How about if it works the second time and this condition then panic?
```if flag.Lookup("test.v") != nil && runMode != libkb.ProductionRunMode```

File: go/teams/loader.go [hunk 1]
```diff
@@ -235,7 +235,11 @@ func (l *TeamLoader) load2InnerLocked(ctx context.Context, arg load2ArgT) (*load
 			l.G().Log.CDebugf(ctx, "Got proof error (%s); trying again with forceRepoll=true", err.Error())
 			arg.forceRepoll = true
 			arg.forceFullReload = true
+			origErr := err
 			res, err = l.load2InnerLockedRetry(ctx, arg)
+			if err == nil {
+				l.G().Log.CDebugf(ctx, "Found an unexpected TeamLodaer case in which busting the cache saved the day (original error was: %s)", origErr.Error())
```
Comment (mlsteele) [inline go/teams/loader.go]: spleling of TeamLoader

**Pull request merged**